.github/workflows/build.yml

```diff
@@ -39,13 +42,64 @@ jobs:
         make -f certifier_tests.mak clean
 
         make -j2 -f certifier.mak
+
+        # We need to clean here, otherwise make certifier_tests.mak will run
+        # into some protobuf-related errors. But this 'clean' will also
+        # rm ../libcertifier_framework.so, so it will need to be re-built.)
         make -f certifier.mak clean
 
         make -j2 -f certifier_tests.mak
         ./certifier_tests.exe
 
+        # Re-build ...; otherwise pylint will complain below.
+        make -f certifier.mak ../libcertifier_framework.so
+
+        popd
+
+        #! Sanity check that the shared library was built correctly and can be loaded
+        #! python3 -c "import libcertifier_framework; help(libcertifier_framework)"
+        #! python3 -c "import libcertifier_tests; help(libcertifier_tests)"
+
+    #! -------------------------------------------------------------------------
+    #! Code formatting runs pylint, so this step can only come after test-py-bindings
+    #! where the shared library (imported by Pytests code) is built.
+    #! -------------------------------------------------------------------------
+    - name: test-code-formatting
+      run: |
+        ./CI/scripts/check-gofmt.sh
+
+        pushd tests
+
+        set -x
+        pylint *.py
+        set +x
+
+        popd
+
+    #! -------------------------------------------------------------------------
+    - name: test-cert_framework-pytests
+      run: |
+        echo " "
+        echo "**************************************************************"
+        echo " Test Python bindings' to Certifier Framework shared library "
+        echo "**************************************************************"
+        echo " "
+
+        pushd tests
+
+        # Run one case showing verbose outputs of each library's contents.
+        # Run each py file separately as 'pytest -v' (on all files) craps out.
+        set -x
+        pytest --capture=tee-sys test_cert_framework_basic.py -k test_getmembers_of_cc_framework
+
+        PYTHONUNBUFFERED=TRUE pytest -v test_cert_framework_basic.py
+
+        PYTHONUNBUFFERED=TRUE pytest --capture=tee-sys -v test_certifier_tests.py
```

Comment:
We need to do this `PYTHONUNBUFFERED=TRUE` stuff so that the output from pytest, reporting each test case being run, is clearly interleaved with output generated by the test itself.